core/src/main/scala/com/mesosphere/usi/core/protos/ProtoBuilders.scala

```diff
@@ -232,17 +232,38 @@ private[usi] object ProtoBuilders {
     b.build()
   }
 
+  def newCommandInfo(shellCommand: String, fetchUri: Seq[FetchUri]): Mesos.CommandInfo = {
+    val b = Mesos.CommandInfo
```

Comment:
it would be great if we used words instead of letters for object references / vals.     It looks like this habit started before this PR...

Reply:
Lol, believe it or not but my method called it `builder` at first and I even started to rename other methods to it but then I realized that if *all* methods call it the same, it's a clear pattern and it doesn't matter whether it is `b` or `builder` (and `b` actually saves you some screen space)

Reply:
I'm really fine with `b` for builders. It's pretty standard and it's not difficult to track in this context. (if there were lots of different builders in scope, or different one-letter variables, I would feel differently)